### ISSUE TITLE
Graphics and buttons missing labels on Home page

### ISSUE THREAD

**Issue by ChrisC** [LLK/scratch-www#7196]
### Expected Behavior

All graphics and interactive elements should have descriptive labels for screen reader users.

### Actual Behavior

The following graphics and buttons on the [Home page](https://scratch.mit.edu/) are missing descriptive labels:

- [ ] Search button in search edit field
- [ ] Graphic in donation message (decorative)
- [ ] Graphic above video explainer (possibly decorative)

We may want to hide purely decorative graphics from the screen reader for a less confusing experience. 

### System Details
Browsers: Chrome/Firefox/Safari, Screen Readers: Jaws/NVDA/VoiceOver

**Screenshots**
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/58723/201795657-7a6ae8ea-1322-4b10-a5fd-f58d8c403d9e.png">

**Comment by ChrisC** [comment]
Since the content on the home page has changed, closing this issue for a more general issue describing issues with missing graphic and button labels on the Home page. See #7265
